flaskfront/opgflaskfront/templates/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
```

Comment:
Remove references to Flask to improve security

Reply:
Done